pkg/service/test/util.go

```diff
@@ -10,10 +11,37 @@ import (
 	"strings"
 	"time"
 
+	"go.keploy.io/server/pkg/hooks"
 	"go.keploy.io/server/pkg/models"
+	"go.keploy.io/server/pkg/platform"
+	"go.keploy.io/server/pkg/platform/yaml"
+	"go.keploy.io/server/pkg/proxy"
 	"go.uber.org/zap"
 )
 
+type InitialiseRunTestSetReturn struct {
+	Tcs           []*models.TestCase
+	ErrChan       chan error
+	TestReport    *models.TestReport
+	DIDE          bool
+	UserIP        string
+	InitialStatus models.TestRunStatus
+	TcsMocks      []*models.Mock
+}
+
+type InitialiseTestReturn struct {
+	SessionsMap              map[string]string
+	TestReportFS             *yaml.TestReport
+	Ctx                      context.Context
+	AbortStopHooksForcefully bool
+	Ps                       *proxy.ProxySet
+	ExitCmd                  chan bool
+	Ys                       platform.TestCaseDB
+	LoadedHooks              *hooks.Hook
+	AbortStopHooksInterrupt  chan bool
+	Ok                       bool
```

Comment:
In the changes ok is used for returning false for errors in starting ap or loading hooks so, instead of boolean we can just return the error along with InitialiseTestReturn for the InitialiseTestReturn.

Reply:
done